integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonCleanFilesCommand.scala

```diff
@@ -41,6 +43,26 @@ case class CarbonCleanFilesCommand(
   extends DataCommand {
 
   val LOGGER: Logger = LogServiceFactory.getLogService(this.getClass.getCanonicalName)
+  val isDryRun: Boolean = options.getOrElse("dryrun", "false").toBoolean
+  var showStats: Boolean = options.getOrElse("statistics", "true").toBoolean
+  if (isInternalCleanCall) {
+    showStats = false
+  }
+
+  override def output: Seq[AttributeReference] = {
+    if (isDryRun) {
+      // dry run operation
+      Seq(
+        AttributeReference("Size Freed", StringType, nullable = false)(),
```

Comment:
better to mention is it in MB/GB

Reply:
is it combining outside trash and inside trash ?

Reply:
I am using convertBytesToReadable before printing it, in that it will add the necessary size value(Kb/Mb etc) in the result itself, so no need to add it in the Attribute Reference. Yes this is both inside and outside trash folder

Reply:
@vikramahuja1001 : can you paste here one output of clean files and dry run now ?

Reply:
@ajantha-bhat 
Clean files with stats: 
+----------+
|Size Freed|
+----------+
|      7 KB|
+----------+

Dry Run:

+----------+--------------------+
|Size Freed|Trash Data Remaining|
+----------+--------------------+
|      7 KB|              0 Byte|
+----------+--------------------+

Reply:
For dry run, 
a) we don't free up the space, so change it to `Size to be freed' 
b) And trash data remaining is only data inside trash right ?  change it to 'Trash folder size' 

for clean files, why are we not printing trash size after cleaning ? @vikramahuja1001 , @akashrn5

Reply:
agree with @ajantha-bhat , change the titles accordingly and @ajantha-bhat for clean files just total cleaned size is fine right? anyways we have separated size in dry run, what do you think?

Reply:
Okay i will change to "Size to be freed". Trash data remaining is both the trash data inside and outside the trash folder